onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -440,6 +442,20 @@ TEST(CApiTest, create_session_without_session_option) {
 }
 #endif
 
+#ifdef REDUCED_OPS_BUILD
```

Comment:
Would be good to add a test that introduces an op (not in the inclusion list) due to optimization and expect a failure in running it.

Reply:
Sounds good, let me see if could compose a model like this.

Reply:
Added model to cover an excluded op - "com.microsoft.BiasGelu".